tools/deb/deb_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+# Test for istio debian. Will run in a docker image where the .deb has been installed.
+
+
+exit 0
```

Comment:
empty file ? what is it doing / adding ?

Reply:
Will be filled in another PR - for now I want to create the docker images and put the infrastructure in place.